packages/core/src/util/hook.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+const iitmHook = require('./iitmHook');
+const requireHook = require('./requireHook');
+// eslint-disable-next-line import/no-useless-path-segments
+const isESMApp = require('./esm').isESMApp;
+
+/*
+ * RequireHook
+ *
+ * Provides support for all CommonJS (CJS) modules in CJS applications, even if the CJS version of
+ * pureEsmLibraries is used in CJS apps.
+ * Also, extends support to all CommonJS (CJS) modules in ESM applications.
+ */
+
+/*
+ * Import-in-the-middle
+ *
+ * Offers support for all native ECMAScript Modules listed in pureEsmLibraries within ESM applications
+ * (including both CJS and ESM versions) of pure ECMAScript Modules.
```

Comment:
> (including both CJS and ESM versions)

When is iitm taking care of cjs code? I cannot follow

Reply:
Removed the confusing comments and improved the readbility.